vql/server/users/grant.go

```diff
@@ -72,6 +73,13 @@ func (self GrantFunction) Call(
 		return vfilter.Null{}
 	}
 
+	services.LogAudit(ctx,
+		org_config_obj, principal, "user_grant",
+		ordereddict.NewDict().
+			Set("Username", arg.Username).
+			Set("Roles", arg.Roles).
```

Comment:
We should probably also log the policy.

This function actually can be used by either providing roles or a policy (which might include roles plus extra permissions)